README.md

```diff
@@ -142,6 +180,52 @@ For example:
 
 The final result of the above command was 10 votes for witness1 and 0 vote for witness2.
 
+## Brokerage
+
+After voting, you will be rewarded with brokerages. Witnesses have the right to decide the ratio of brokerages. The default ratio is 20%, and witnesses can adjust it.
```

Comment:
After voting for the witness, you will receive the rewards. The witness has the right to decide the ratio of brokerage.